libs/pulumi/src/generators/destroy-stack/generator.ts

```diff
@@ -59,6 +59,12 @@ export default async function (
             stdio: [process.stdin, process.stdout, process.stderr],
         })
 
+        // remove the pending operations from the state
+        updateJson(tree, './state.json', (state) => {
+            delete state.deployment.pending_operations
```

Comment:
Is it safe to remove all pending CRUD operations?

Update and Delete are probably safe to remove, but we (probably) need to throw an error on pending Create operations

Reply:
(so we can manually clean up the resources)